CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable, unreleased changes to this project will be documented in this file.
 # 3.15.0 [Unreleased]
 
 ### Breaking changes
+- Use public key thumbprint as KID in JWKS.json #13442 by @cmiacz
```

Comment:
If this is a breaking change, we need to provide more context e.g. who is affected and how they should migrate to the new version. E.g. we can add a migration guide similar to this: https://docs.saleor.io/docs/3.x/upgrade-guides/3-13-to-3-14

Reply:
It's not breaking chane, but fix for unexpected behavior. I've fixed the changelog file.